src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -188,6 +196,9 @@ class MessageSection extends Component {
   constructor(props) {
     super(props);
     this.state = getStateFromStores();
+    this.getMessageListItem = getMessageListItem.bind(this);
```

Comment:
No need to bind explicitly, use arrow function

Reply:
the getMessageListItem is a declared function and not an arrow function, I don't see how to avoid using a bind here

Reply:
Can we make it work by removing `this` at line 643, 645?